eng/versioning/version_client.txt

```diff
@@ -5,7 +5,7 @@ com.azure:azure-sdk-all;1.0.0;1.0.0
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-formrecognizer;1.0.0-beta.2;1.0.0-beta.3
-com.azure:azure-ai-textanalytics;1.0.0-beta.5;1.0.0-beta.6
+com.azure:azure-ai-textanalytics;1.0.0-beta.5;1.0.0
```

Comment:
should this happen after we actually ship?

Reply:
the section of 1.0.0-beta.6 is under `current` version. If we released, 'dependency` version will be change to 1.0.0.

Reply:
But this should still happen after the release?
cc: @JimSuplizio

Reply:
@maririos @mssfang @samvaity 
It would depend, 1.0.0-beta.5 is current released version of com.azure:azure-ai-textanalytics which is why it's the _dependency_ version. 1.0.0-beta.6 is/was the _current_ version, if the version of this library being released is GA version 1.0.0, then this change is fine. You set the release version to what it needs to be just prior to release. After release, the 1.0.0 would become the new _dependency_ version and the new _current_ version would be 1.1.0-beta.1, that is the PR that the build system would create. If the version being released were beta then we'd tick up the beta number automatically as part of the release process. beta.x->beta.x+1.